### PR TITLE
test(table): added unit tests

### PR DESCRIPTION
**Pull request by alfredskeden**
**Describe pull-request**  
Followed the test steps in ticket/excel to test.

**Solving issue**  
Fixes: [CDEP-2962](https://tegel.atlassian.net/browse/CDEP-2962)

**How to test**  
1. Navigate to packages/core.
2. Check the code for tests in table component.
3.  Make sure docker is running.
4. Then run `npm run test` in core to run the tests. Make sure they run.

**Additional context** 
We are still in exploration for good testing.

[CDEP-2962]: https://tegel.atlassian.net/browse/CDEP-2962

### CODE REVIEW

File: packages/core/src/components/table/table/test/basic/table.e2e.ts [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+  test('Row should contain the correct number of rows with', async ({ page }) => {
+    await page.goto(componentTestPath);
+
+    /* Expect the number of rows to be correct amount */
+    const tableBodyRows = page.locator('tds-table-body-row');
+    await expect(tableBodyRows).toHaveCount(6);
+  });
+
+  test('table the correct text inside each cell', async ({ page }) => {
```
Comment (alexandereneroth) [inline packages/core/src/components/table/table/test/basic/table.e2e.ts:40]:
```suggestion
  test('table has the correct text inside each cell', async ({ page }) => {
```

File: packages/core/src/components/table/table/test/basic/index.html [hunk 2]
```diff
@@ -0,0 +1,162 @@
...
+    <meta name="viewport" content="width=device-width, initial-scale=1.0, minimum-scale=1.0" />
+    <link rel="stylesheet" href="../../../../../../dist/tegel/tegel.css" />
+    <script type="module">
+      import { defineCustomElements } from '../../../../../../loader/index.es2017.js';
+      defineCustomElements();
+    </script>
+  </head>
+
+  <body>
+    <tds-table vertical-dividers="false" compact-design="false" responsive="false">
```
Comment (alexandereneroth) [inline packages/core/src/components/table/table/test/basic/index.html:15]: I think in the dropdown the basic table had no properties. I suggest removing the properties or renaming it to default (I think thats what it was called in dropdown). WDYT?

**Review by alexandereneroth — APPROVED** [review]
Looks good just left a couple comments for some minor things

**Review by timrombergjakobsson — APPROVED** [review]
Nice work!!

**Pull request merged**